### PR TITLE
Fix bugs 2

### PR DESCRIPTION
**Pull request by ali-7**
relates to #70

### CODE REVIEW

**Review by lubnaabd — APPROVED** [review]

**Review by HemaSAli — APPROVED** [review]

File: public/css/mainStyleWebSite.css [hunk 1]
```diff
@@ -205,8 +205,13 @@ h5{
   padding-right: 0.5rem;
 }
 
-.icons img{
+.icons img {
   margin: 1%;
```
Comment (NouraldinS) [inline public/css/mainStyleWebSite.css]: Remove relative margin.

File: src/views/cohort.hbs [hunk 2]
```diff
@@ -19,10 +19,10 @@
         <td>{{name}}</td>
         <td>{{description}}</td>
         <td>
-          <a href="{{githublink}}">click</a>
+          <a href="{{githublink}}" target="_blank">click</a>
```
Comment (NouraldinS) [inline src/views/cohort.hbs]: Replace with `githubLink`.

File: src/views/cohort.hbs [hunk 3]
```diff
@@ -19,10 +19,10 @@
...
         </td>
         <td>
-          <a href="{{img_url}}">click</a>
+          <a href="{{img_url}}" target="_blank">click</a>
```
Comment (NouraldinS) [inline src/views/cohort.hbs]: Replace with `imgUrl`.

File: src/views/cohortStudents.hbs [hunk 4]
```diff
@@ -19,8 +19,8 @@
       <tr>
         <td>{{name}}</td>
         <td>{{username}}</td>
-        <td><a href="{{githublink}}">Click</a></td>
-        <td><a href="{{img_url}}">Click</a></td>
+        <td><a href="{{githublink}}" target="_blank">Click</a></td>
```
Comment (NouraldinS) [inline src/views/cohortStudents.hbs]: Replace with `githubLink`.

File: src/views/cohortStudents.hbs [hunk 5]
```diff
@@ -19,8 +19,8 @@
...
+        <td><a href="{{img_url}}" target="_blank">Click</a></td>
```
Comment (NouraldinS) [inline src/views/cohortStudents.hbs]: Replace with `imgUrl`

File: src/views/cohortStudents.hbs [hunk 6]
```diff
@@ -19,8 +19,8 @@
...
         <td>
           <a class="edit" href = "/admin/cohorts/{{cohort_id}}/editStudent/{{studentId}}"id="{{studentId}}">Edit</a>
```
Comment (NouraldinS) [inline src/views/cohortStudents.hbs]: Replace with `cohortId`.

File: src/views/projectPageWebsite.hbs [hunk 7]
```diff
@@ -12,14 +12,14 @@
   </h1>
   <ul class="information-ui">
     <li> {{res.projectResult.0.cohort_name}} </li>
```
Comment (NouraldinS) [inline src/views/projectPageWebsite.hbs]: Do not send the entire response to handlebars and deal with it later, send only the cohort name as `cohortName` and whatever you need out of the result.

File: src/views/projectPageWebsite.hbs [hunk 8]
```diff
@@ -12,14 +12,14 @@
...
     <li>
       students
       <ul class="sutdentsUl">
         {{#if res.projectStudentsResult.length}}
           {{#each res.projectStudentsResult}}
-           <li><a target="_blank" href="{{github_link}}"> {{studentName}}</a></li>
+           <li><a target="_blank" href="{{githubLink}}"> {{studentName}}</a></li>
           {{/each}}
         {{else}}
           <h3>There are No Student !</h3>
```
Comment (NouraldinS) [inline src/views/projectPageWebsite.hbs:22]: Fix indentation.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

**Review by NouraldinS — APPROVED** [review]

**Review by NouraldinS — CHANGES_REQUESTED** [review]
Solve all mentioned problems.

File: src/views/projectPageWebsite.hbs (hunk 8 above)
Comment (ali-7) [reply src/views/projectPageWebsite.hbs:22]: @NoureldeanSaed which line this indentation problem ??!! i can't find it !!

**Review by NouraldinS — APPROVED** [review]

**Pull request merged**